resources/js/provider.js

```diff
@@ -188,7 +188,12 @@
         if (!this._events[name]) {
           return
         }
-        this._events[name].forEach(cb => cb(data));
+        this._events[name].forEach(cb => {
+            try{
+                cb(data)
+            } catch(e){
+                setTimeout((()=>{throw e}))
+            }});
```

Comment:
I guess you are using `setTimeout` to throw the error, so that it shouldn't stop the current js thread. Please can you explain, what is the benefit of doing that, can we just use an empty block here?